com.unity.render-pipelines.core/Editor/Converter/RenderPipelineConvertersEditor.cs

```diff
@@ -0,0 +1,436 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEditor;
+using UnityEditor.Rendering;
+using UnityEditor.UIElements;
+using UnityEngine.UIElements;
+
+// Status for each row item to say in which state they are in.
+// This will make sure they are showing the correct icon
+enum Status
+{
+    Pending,
+    Warning,
+    Error,
+    Success
+}
+
+// This is the serialized class that stores the state of each item in the list of items to convert
+[Serializable]
+class ConverterItemState
+{
+    public bool isActive;
+
+    // Message that will be displayed on the icon if warning or failed.
+    public string message;
+
+    // Status of the converted item, Pending, Warning, Error or Success
+    internal Status status;
+
+    internal bool hasConverted = false;
+}
+
+// Each converter uses the active bool
+// Each converter has a list of active items/assets
+// We do this so that we can use the binding system of the UI Elements
+[Serializable]
+class ConverterState
+{
+    // This is the enabled state of the whole converter
+    public bool isEnabled;
+    public bool isActive;
+    public bool isInitialized;
+    public List<ConverterItemState> items;
+
+    public int pending;
+    public int warnings;
+    public int errors;
+    public int success;
+    internal int index;
+}
+
+[Serializable]
+[EditorWindowTitle(title = "Render Pipeline Converters")]
+public class RenderPipelineConvertersEditor : EditorWindow
+{
+    Texture2D kImgWarn;
+    Texture2D kImgHelp;
+    Texture2D kImgFail;
+    Texture2D kImgSuccess;
+    Texture2D kImgPending;
+
+    public VisualTreeAsset m_ConverterEditorAsset;
```

Comment:
if they are public they can not start with `m_`

Reply:
done